msal/src/Microsoft.Identity.Client/Features/IntegratedWindowsAuth/IntegratedWindowsAuthRequest.cs

```diff
@@ -120,7 +127,7 @@ private async Task UpdateUsernameAsync()
             }
         }
 
-        private Dictionary<string, string> GetAdditionalBodyParameters(UserAssertion userAssertion)
+        private static Dictionary<string, string> GetAdditionalBodyParameters(UserAssertion userAssertion)
```

Comment:
>static [](start = 16, length = 6)

is there a way to make this not a static?

Reply:
Let's not demonise static :). Static methods are faster than instance methods, but more importantly, they make the code cleaner when there is no state involved. In this case, the state is represented by the 2 members of the class (_iwaInput and _common...) plus the many members of the base class (yet another reason why inheritance is frowned upon - it hides state). 

This method GetAdditionalBodyParameters does not touch the state, because we did not make userAssertion a member of this class. In fact, R# tells you that a method can be made static for this reason.